### ISSUE TITLE
3D Encoder Evaluation on ModelNet40

### ISSUE THREAD

**Issue by TheNeeloy** [jetd1/condense#4]
Thanks for your great work!

I am attempting to evaluate the 3D encoder on the classification task on ModelNet40.
However, I ran into issues when actually running the code - some of which are listed below:

1. The ModelNet40 dataset from ULIPv2 (which I believe you are using) does not include CLIP text embeddings for the classes
- And I am unsure of which OpenCLIP model was used to get the features in the ConDense repo for top-K retrieval accuracy evaluation

2. The released code for the 3D encoder does not include any config files for the experiments
- I wrote one from scratch based on the keys that the eval code refers to, but I am unsure if it is correct

3. The 3D classification accuracy code seems to only attempt to load weights from one path in the config
- But, my understanding is that the model should be loading both the pretrained backbone and the MLP head for the specific dataset to evaluate on

4. Since the config file is not released, I am unsure of how many output channels the 3D encoder in the eval code should have

5. I also printed out the state dict keys of both the 3D encoder backbone and the ModelNet40 head checkpoints and compared with the state dict keys of a newly initialized 3D encoder model in the evaluation code
- The head weights seems to output 1000 logits even though ModelNet40 has only 40 classes
- None of the keys seem to match between the backbone checkpoint and a freshly initialized 3D encoder model's state dict

If possible, I would truly appreciate your input on this matter. Would there be more finalized code I could use to evaluate the ConDense model?
Please feel free to let me know if I can provide any additional information.

Best,
Neeloy
